scripts/ci/bump_version.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+# Fetch all tags
+git fetch --tags
+
+# Get the latest tag (sorted by creation date)
+LATEST_TAG=$(git tag --sort=-creatordate | head -n 1)
+echo "Latest tag: $LATEST_TAG"
+
+# Remove  .Final suffix if present
+BASE_VERSION=${LATEST_TAG%.Final}
+
+echo "Previous version: $BASE_VERSION"
+
+echo "prev_ver=$BASE_VERSION" >> "$GITHUB_OUTPUT"
+
+IFS='.' read -r MAJOR MINOR PATCH <<< "$BASE_VERSION"
+
+# This logic works for 2.4.x tags, we can tweak it later.
+if [ "$PATCH" -ge 18 ]; then
```

Comment:
Now that 8.5.0 is out, this no longer makes sense. If the release is triggered manually we could add an input option for a new minor. (It'd reset patch and increased minor)

Reply:
removed the logic